go/protocol/identify.go

```diff
@@ -1,4 +1,4 @@
-// Auto-generated by avdl-compiler v1.1.1 (https://github.com/keybase/node-avdl-compiler)
+// Auto-generated by avdl-compiler v1.2.0 (https://github.com/keybase/node-avdl-compiler)
```

Comment:
Sure

On Wed, Mar 23, 2016 at 3:49 PM, Patrick Crosby <notifications@github.com>
wrote:

> In go/protocol/identify.go
> <https://github.com/keybase/client/pull/2433#discussion_r57225100>:
>
> > @@ -1,4 +1,4 @@
> > -// Auto-generated by avdl-compiler v1.1.1 (https://github.com/keybase/node-avdl-compiler)
> > +// Auto-generated by avdl-compiler v1.2.0 (https://github.com/keybase/node-avdl-compiler)
>
> then shouldn't npm i be in the makefile as a dependency?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2433/files/8b973c73962ef0e9b187eccd2b5be034e6520136#r57225100>
>